Makefile

```diff
@@ -0,0 +1,25 @@
+VIRTUALENV = virtualenv
+VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+PYTHON = $(VENV)/bin/python
+TOX = $(VENV)/bin/tox
+TEMPDIR := $(shell mktemp -d)
+
+build-requirements:
+	$(VIRTUALENV) $(TEMPDIR)
+	$(TEMPDIR)/bin/pip install -U pip
+	$(TEMPDIR)/bin/pip install -Ue .
+	$(TEMPDIR)/bin/pip freeze > requirements.txt
```

Comment:
Can we use the improved version of it? See https://github.com/Kinto/kinto/blob/master/Makefile#L64-L68